PdCore/build.gradle

```diff
@@ -31,5 +34,24 @@ android {
         }
 
         instrumentTest.setRoot('tests')
-    }
+    }	
+}
+
+dependencies {
+    compile project(':midi:AndroidMidi')
```

Comment:
This line is not working for me, but is working with

    compile project(':AndroidMidi')

May be related to gradle version 1.0.0 (see other comment).